text2text/identifier.py

```diff
@@ -0,0 +1,50 @@
+import os
+import requests
+import pickle
+import text2text as t2t
+
+
+class Identifier():
```

Comment:
```suggestion
class Identifier(t2t.Vectorizer):
```

Reply:
Hi Art @artitw  I initially inherited from the Vectorizer, but it gave me this error: `AttributeError: type object 'Identifier' has no attribute 'tokenizer'`, hence the reason I removed it. 

I'm thinking the identifier class has to have the attributes the vectorizer class has, but is there a way to bypass this?

Reply:
Thanks for checking on that. It looks like the issue is that we need to inherit and then also override the constructor, so an extra line of code is necessary to initialize from `t2t.Vectorizer` first. See https://github.com/artitw/text2text/pull/32/files#r846677457